clang/test/CodeGen/aarch64-sme-inline-streaming-attrs.c

```diff
@@ -0,0 +1,47 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -S -target-feature +sme -verify -DTEST_NONE -x c %s
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -S -target-feature +sme -verify -DTEST_COMPATIBLE -x c %s
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -S -target-feature +sme -verify -DTEST_STREAMING -x c %s
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -S -target-feature +sme -verify -DTEST_LOCALLY -x c %s
```

Comment:
If you use `-fsyntax-only` then you don't need to use the `#define`'s.

Reply:
Unfortunately the part of the code that checks this isn't run under -fsyntax-only. It's the same for the existing target feature checks with always_inline: https://godbolt.org/z/frrbeah7d